src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -441,9 +442,9 @@ export class CellExecution implements IDisposable {
         const props = { notebook: true };
         if (!CellExecution.sentExecuteCellTelemetry) {
             CellExecution.sentExecuteCellTelemetry = true;
-            sendTelemetryEvent(Telemetry.ExecuteCellPerceivedCold, this.stopWatch.elapsedTime, props);
```

Comment:
Using the `stopWatch` property is wrong.
This value gets reset just when the cell is sent to Jupyter. Thus this isn't the perceived time
E.g. when usre opens a notebook an hits run, and kernel takes 10s to start, afer it starts we send this cell to Jupyter and re-set the stopwatch, but now we're sending <10 for execution times.

Hence using a new property to capture the perceived times.